### DISCUSSION TITLE
Scroll depth not working for sharepoint online for scrollable Div elements

### DISCUSSION THREAD

**Discussion by borrelnoten** [vmware/clarity#5671] [Q&A]
In sharepoint online, content is placed in divs which are scrollable.

`<div data-is-scrollable="true" class="a_a_beed2cf1 c_a_beed2cf1" data-automation-id="contentScrollRegion" role="main" tabindex="-1" data-sp-component-lazycount="-10">
`
The scroll maps seem to only record actual browser window scrolling and not scrolling within divs.

**Comment by hippee-lee** [comment]
@borrelnoten  - this seems like a question so please correct me if I am wrong. 

I converted it to a discussion so we could flesh out a few more things:

1) a simple reproduction with code that we can point to for triage and investigation
2) If this is a bug report, please add the [requested details  ](https://github.com/vmware/clarity/blob/next/.github/ISSUE_TEMPLATE/bug-report.md)
3) wrt sharepoint - This isn't a standard use case for employing a Clarity application so more details about the use case and way it has been created will help drive the conversation. 

Clarity has not been tested inside of content management systems like sharepoint but I suspect that there could be many css issues that occur. It is hard to help without enough information, please take a few minutes to create the simple reproduction, share the source code and write down the details.
